ci-operator/step-registry/baremetalds/sno/rename/baremetalds-sno-rename-commands.sh

```diff
@@ -228,6 +228,9 @@ then
 
   delete_crts_keys
 
+  echo "Removing previous OVN dbs..."
+  rm -rf /var/lib/ovn-ic/etc/ovn*.db
```

Comment:
@tsorya should we add this to the installation-reconfiguration script?

Reply:
It is part of the seed creation 
https://github.com/openshift-kni/lifecycle-agent/blob/main/lca-cli/seedcreator/seedcreator.go#L431